applications/nrf_desktop/src/modules/dfu.c

```diff
@@ -143,6 +162,15 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_LOG_LEVEL);
 	#error Bootloader not supported.
 #endif
 
+/* This value denotes whether the revert feature is supported by the bootloader.
+ * Currently, the revert feature is only supported by the MCUboot bootloader that
+ * is not configured for the standard direct-xip or RAM load mode.
+ */
+#define DFU_REVERT_FEATURE_IS_SUPPORTED                            \
```

Comment:
`MCUBOOT_REVERT_FEATURE_IS_SUPPORTED` (as we refer to MCUboot's revert feature)

Reply:
Or maybe even `MCUBOOT_IMAGE_CONFIRM_NEEDED` (to indicate that we need to confirm the image after it's uploaded)